samples/subsys/ipc/openamp/common.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ * Copyright (c) 2018 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef COMMON_H__
+#define COMMON_H__
+
+#define SHM_START_ADDR		0x04000400
+#define SHM_SIZE		0x7c00
+#define SHM_DEVICE_NAME		"sramx.shm"
+
+#define VRING_COUNT		2
+#define VRING_RX_ADDRESS	0x04007800
+#define VRING_TX_ADDRESS	0x04007C00
+#define VRING_ALIGNMENT		4
+#define VRING_SIZE		16
+
+#define RSC_TABLE_ADDRESS	0x04000000
```

Comment:
Should the resource table address covered by the shm? as this is beyond the SHM_START_ADDR, just wonder where is this memory mapped?

Reply:
What do you mean by mapped?

Reply:
@wjliang ?